### DISCUSSION TITLE
Anilist 6.0.0.0 can't be uninstalled...

### DISCUSSION THREAD

**Discussion by shimizurei** [jellyfin/jellyfin#12474] [Q&A] [answered]
My AniList 6.0.0.0 won't uninstall when I hit "uninstall" on the web UI (it just shows an endless loader) and there is no associated folder?

![image](https://github.com/user-attachments/assets/6bd3eb1b-8fab-4e2c-a603-d5ae09617d33)

Also, can I remove the older versions for my other plugins from the plugin folder? Not why some have the old versions and some don't.

![image](https://github.com/user-attachments/assets/8a1a2137-210c-487e-8e24-19fc48ae7be3)

**Comment by sjorge** [comment, marked as answer]
One of the AniList plugin releases had a wrong manifest where one of the versions wasn't updated.

You should be able to just remove those plugin folders if you stop jellyfin before.

**Reply by shimizurei** [reply]
Thanks. I removed the old folders with no issue.

For AniList, I stopped my container, removed ALL the AniList plugin folders, restarted the container, and re-installed the plugin. This removed the ghost plugin.
